src/components/Accordion/__snapshots__/Accordion.stories.storyshot

```diff
@@ -59,13 +59,7 @@ exports[`Storyshots Core/Accordion Default 1`] = `
           <div
             aria-hidden="true"
             class="StyledBox-sc-13pk1d4-0 dYebPD Collapsible__AnimatedBox-sc-15kniua-0 euNsBL"
-          >
```

Comment:
This doesn't look right, can you check if something broke? It might have to do with the `setupTests` changes.

Reply:
I reverted setupTests change and tried again, snapshots are updated anyway. I suspect that this is due to change in grommet's `GrommetAccordionPanel` which does not hide panel any more, instead it does not render html at all: https://github.com/grommet/grommet/pull/4701/files

Reply:
@karaxuna makes sense, thanks!